Makefile

```diff
@@ -1,3 +1,9 @@
+GO_PKG=github.com/openshift/insights-operator
+GO111MODULE?=on
+GOFLAGS=-mod=vendor
+export GO111MODULE
+export GOFLAGS
+
```

Comment:
I don't believe these are needed, I had the same on one of my backports

Reply:
Now that you pointed it all, I think that the only thing that makes sense to keep is the: `GO111MODULE?=on`

Reply:
Or maybe not even this one... o_O